logprep/util/defaults.py

```diff
@@ -1,5 +1,18 @@
 """Default values for logprep."""
 
+from enum import Enum
+
+
+class EXITCODES(Enum):
+    """Exit codes for logprep."""
+
+    SUCCESS = 0
+    ERROR = 1
+    CONFIGURATION_ERROR = 2
+    PIPELINE_ERROR = 3
+
+
+DEFAULT_RESTART_COUNT = 5
```

Comment:
should we make this configurable? I think it would be nice if the end user could configure how forgiving logprep should be? Like restart_count of 0 would mean logprep stops after every error, and restart_count = 100 could mean something like never give up. And that "never give up" could even actually be implemented with something like "-1" maybe.